packages/app/src/queries/staking/utils.ts

```diff
@@ -59,8 +62,15 @@ export function getApy(totalStakedBalance: number, weekCounter: number) {
 
 export const parseEpochData = (index: number, networkId?: NetworkId) => {
 	const { epochStart, epochEnd } = getEpochDetails(networkId ?? 10, index)
-	const startDate = formatShortDate(new Date(toJSTimestamp(epochStart)))
-	const endDate = formatShortDate(new Date(toJSTimestamp(epochEnd)))
-	const label = `Epoch ${index}: ${startDate} - ${endDate}`
+	const label = `Epoch ${index}`
 	return { period: index, start: epochStart, end: epochEnd, label }
 }
+
+export const useEstimatedReward = (fileName: string) => {
+	const network = useAppSelector(selectNetwork)
+	const walletAddress = useAppSelector(selectWallet)
+	const query = useGetFile(
```

Comment:
I think we can move this out of utils as it's own query hook. Also ideally we move away from these useQuery hooks and use the async functions in redux / store in state but can look at that as a refactor.

Reply:
Good idea. Updated.